_studio/shared/umc/core/umc/include/umc_defs.h

```diff
@@ -33,20 +33,7 @@
 
 #if defined(LINUX32) && !defined(ANDROID)
 
-    // video decoders
-    #define MFX_ENABLE_VC1_VIDEO_DECODE
-    #define MFX_ENABLE_MJPEG_VIDEO_DECODE
-    #define MFX_ENABLE_MPEG2_VIDEO_DECODE
-    #define MFX_ENABLE_H264_VIDEO_DECODE
-    #define MFX_ENABLE_H265_VIDEO_DECODE
-    #define MFX_ENABLE_VP8_VIDEO_DECODE
-    #define MFX_ENABLE_VP9_VIDEO_DECODE
-
-    // video encoders
-    #define MFX_ENABLE_MVC_VIDEO_ENCODE
-    #define MFX_ENABLE_MJPEG_VIDEO_ENCODE
-    #define MFX_ENABLE_MPEG2_VIDEO_ENCODE
-    #define MFX_ENABLE_H264_VIDEO_ENCODE
+#include "mfx_config.h"
```

Comment:
this is part of the previous patch! By the way, I very much doubt you need if-defined-LINUX-ANDROID...

Reply:
And we had 2 different sets of android defs! First in umc_defs.h and second in mfx_android_defs.h!